src/plugins/kibana_usage_collection/server/collectors/management/types.ts

```diff
@@ -73,6 +73,7 @@ export interface UsageStats {
   'discover:searchOnPageLoad': boolean;
   'doc_table:hideTimeColumn': boolean;
   'discover:sampleSize': number;
+  'discover:sampleRowsPerPage': string;
```

Comment:
nit: is it a `string` or a `number`?

Reply:
@afharo Currently, a Select UI element gives values only as strings. We are discussing it now whether we should rather switch to a simple input element on Advanced Settings page.

Reply:
@afharo Okay, we are keeping the Select UI element. 

I updated [onFieldChange](https://github.com/elastic/kibana/pull/135726/files#diff-e105abaf054b13383ef9752fd5f573d15f6028b8428adb72fa9e51c805484ccbR173) handler to return values as numbers if `options` included numbers and changed the type of `discover:sampleRowsPerPage` to number . Wdyt?